tutorial/cloudshell_tutorial.md

```diff
@@ -167,7 +167,7 @@ Add the following to your main.tf file:
 
 ```hcl
 resource "google_project_services" "project_services" {
-  project  = "<PROJECT_ID>"
+  project  = "{{project-id}}"
```

Comment:
This directive will expand into the project set before.

![image](https://user-images.githubusercontent.com/2080520/63113825-43597f00-bf48-11e9-8ada-29b10b65cf10.png)